frontend/src/app/commonComponents/FormGroup.tsx

```diff
@@ -1,19 +1,24 @@
 import React from "react";
 
-interface Props {
+interface FormGroupProps {
   title: string;
   children: React.ReactNode;
+  asSectionGroup?: boolean;
 }
 
-const FormGroup = (props: Props) => (
-  <form className="prime-formgroup">
-    <fieldset className="usa-fieldset">
-      <legend className="prime-formgroup-heading usa-legend">
-        {props.title}
-      </legend>
-      {props.children}
-    </fieldset>
-  </form>
-);
+const FormGroup = (props: FormGroupProps) => {
+  const WappringEl = props.asSectionGroup ? "div" : "form";
```

Comment:
I think this is a typo? Should it be `WrappingEl` instead?

Reply:
omg yes! thanks for catching this :D